tests/unit_tests/pyanaconda_tests/modules/timezone/test_module_timezone.py

```diff
@@ -587,3 +597,44 @@ def _validate_ntp_config(self, sysroot, was_present, was_enabled, expected_lines
 
         elif not was_present:
             assert not os.path.exists(sysroot + NTP_CONFIG_FILE)
+
+
+class TimezoneHardwareClockTasksTestCase(unittest.TestCase):
+    """Test the D-Bus Timezone Hardware Clock task."""
+
+    @patch("pyanaconda.modules.timezone.installation.util.execWithRedirect")
+    @patch('pyanaconda.modules.timezone.installation.arch.is_s390', return_value=True)
+    def test_hwclock_config_task_s390(self, mock_is_s390, mock_exec_with_redirect):
+        """Test that save_hw_clock does nothing on s390."""
+        self._execute_task(False)
+        # expected state: calling it only once in the check for architecture
+        mock_is_s390.assert_called_once()
+        mock_exec_with_redirect.assert_not_called()
+
+    @patch("pyanaconda.modules.timezone.installation.util.execWithRedirect")
+    def test_hwclock_config_task_local(self, mock_exec_with_redirect):
+        """Test the Hardware clock configuration task - local"""
+        with patch("pyanaconda.modules.timezone.installation.conf") as core_util:
+            core_util.system.can_set_hardware_clock(True)
```

Comment:
I think that `conf.system.can_set_hardware_clock` still returns the default mock object after this call. Try: `core_util.system.can_set_hardware_clock = True`

Reply:
Btw. you should be able to add a test for `conf.system.can_set_hardware_clock == False` now.